ckanext-offgridtheme/ckanext/offgridtheme/static/scripts/maps.js

```diff
@@ -0,0 +1,52 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoiZG1vcmlhcnR5IiwiYSI6Ikd3T29EOWMifQ.-DKJ4ernht84AZmc6Bk51Q';
+var map_1 = new mapboxgl.Map({
+    container: 'ssa-aicd-electricity-map',
+    style: '/data/map-style.json',
```

Comment:
Can we use Mapbox's `light-v9` for this? It's being used in the other map previews.